### DISCUSSION TITLE
Mozilla Firefox changes input's placeholder style

### DISCUSSION THREAD

**Discussion by simasjar** [iTwin/iTwinUI#1417] [Q&A] [answered]
Hello,

I recently found that the input placeholder's opacity changes when the browser is Mozilla Firefox.

For example, open [this sandbox](https://codesandbox.io/s/elastic-tristan-qkdgtw?file=/src/App.tsx) in Mozilla Firefox and we can see that the placeholder's style differs from the custom div's style, though I am using the same color from the input placeholder.
![image](https://github.com/iTwin/iTwinUI/assets/124659623/7043fdaf-fed2-4d11-acd4-ba86a3d64891)

If the sandbox is opened in another browser (for example, Edge), the placeholders look the same: 
![image](https://github.com/iTwin/iTwinUI/assets/124659623/85546a19-7dde-42a4-b93d-468b23b7e9ca)

Is this the expected behavior?

Also, after adding this CSS class: 

```css
input::placeholder {
  opacity: 1;
}
```
placeholders look the same in Mozilla Firefox.

**Comment by mayank99** [comment, marked as answer]
looks like this is documented behavior in firefox. https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder#opaque_text

does it cause any usability issues? placeholder text should not be used for critical information (that is what the label should be used for), so i wouldn't mind just leaving it as-is.

**Reply by mayank99** [reply]
update: we fixed it after all in ^2.12.3
